sources/platform/console/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Apify Console
+description: Learn about Apify Console's easy account creation and user-friendly homepage for efficient web scraping management.
+sidebar_position: 1
+category: platform
+slug: /console
+---
+
+**Learn about Apify Console's easy account creation and user-friendly homepage for efficient web scraping management.**
+
+---
+
+## Sign Up
+
+To use Apify Console, you need to create Apify account. [Sign up here](https://console.apify.com/sign-up).
+
+![apify-console-sign-up](./images/console-signup.png)
+
+## Log In
+
+To log in to Apify Console, you can use one of the following methods:
+
+* Google Single Sign-On
+* GitHub Single Sign-On
+* The email and password set during account creation
+
+![apify-console-login](./images/console-login.png)
+
+## Homepage Overview
+
+![apify-console-homepage](./images/console-homepage.png)
+
+The Apify Console homepage provides an overview of our services. The header displays your account name and current plan level. The homepage features several sections:
```

Comment:
Suggestion: I'd go with something like this instead, it's not showing our services, but the stuff relevant to the user.

```suggestion
The Apify Console homepage provides an overview of your account setup. The header displays your account name and current plan level. The homepage features several sections:
```

Also note that the homepage will probably also change quite a lot quite soon.